### PR TITLE
feat: better FR translation

### PR DESCRIPTION
**Pull request by lahgolz**
Translations from my French version of the blockchain explorer before the release of the official version by the Core Team.

### CODE REVIEW

**Review by rolanasser — APPROVED** [review]

**Review by yusopwd — APPROVED** [review]

File: src/languages/fr.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
 {
   "account": "Compte",
   "accounts": "Comptes",
-  "account.id": "ID du Compte",
```
Comment (Guqson) [inline src/languages/fr.json:4]: Account.Id" : Id  core computer "

**Review by Guqson — APPROVED** [review]

**Review by Nazirul108 — APPROVED** [review]

**Comment by V1e2n3k4a5t6e7s8w9a10r11l12u13** [comment]
Ok

**Review by Robin8004 — APPROVED** [review]

**Comment by V1e2n3k4a5t6e7s8w9a10r11l12u13** [comment]
Ok

**Comment by fakhroshumailasabri786** [comment]
Halla

**Pull request closed without merging**